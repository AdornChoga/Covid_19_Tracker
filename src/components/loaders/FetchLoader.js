import React from 'react';
import ClipLoader from 'react-spinners/ClipLoader';
import { css } from '@emotion/react';
import classes from './FetchLoader.module.css';

const FetchLoader = () => {
  const override = css`
    border-color: white;
  `;

  return (
    <div className={classes.loader_container} data-testid="loading_spinner_page">
      <ClipLoader loading css={override} size={100} />
      <h3>Fetching Covid-19 Statistics...</h3>
    </div>
  );
};

export default FetchLoader;
