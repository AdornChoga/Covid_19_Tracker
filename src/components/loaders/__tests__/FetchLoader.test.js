import React from 'react';
import renderer from 'react-test-renderer';
import FetchLoader from '../FetchLoader';

it('is loading spinner page snapshot created', () => {
  const tree = renderer.create(<FetchLoader />).toJSON();
  expect(tree).toMatchSnapshot();
});
