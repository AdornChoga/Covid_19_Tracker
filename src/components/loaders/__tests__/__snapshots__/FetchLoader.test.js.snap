// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`is loading spinner page snapshot created 1`] = `
<div
  className="loader_container"
  data-testid="loading_spinner_page"
>
  <span
    className="css-9i2kbp"
  />
  <h3>
    Fetching Covid-19 Statistics...
  </h3>
</div>
`;
